README.md

```diff
@@ -1,7 +1,7 @@
 [![Build Status](https://travis-ci.org/OCA/account-financial-reporting.svg?branch=10.0)](https://travis-ci.org/OCA/account-financial-reporting)
 [![Coverage Status](https://coveralls.io/repos/OCA/account-financial-reporting/badge.png?branch=10.0)](https://coveralls.io/r/OCA/account-financial-reporting?branch=10.0)
 
-OpenERP account financial reports
+Odoo account financial reports
 =================================
```

Comment:
Please change the length of this line to have the same as the title

Reply:
Done